mobile/app/login.tsx

```diff
@@ -43,6 +46,14 @@ const Login = () => {
       <Typography preset="body1">body1</Typography>
       <Typography preset="body2">body2</Typography>
       <Typography preset="helper">helper</Typography>
+
+      <Card
+        children={
```

Comment:
not an issue right now but keep in mind this is not how we usually pass children to components
`<Card>
      <Text>...</Text> // this is automatically a child of Card now
  </Card>
`

Reply:
done